fabtests/scripts/runfabtests.sh

```diff
@@ -318,7 +318,7 @@ function unit_test {
 	start_time=$(date '+%s')
 
 	cmd="${BIN_PATH}${test_exe}"
-	${SERVER_CMD} "$cmd" &> $s_outp &
+	${SERVER_CMD} "${EXPORT_ENV}$cmd" &> $s_outp &
```

Comment:
nit: let's add a space between the two variables and skip it when setting EXPORT_ENV

Reply:
Sure on the space comment. What is being skipped?

Reply:
I meant adding space here could enable skipping adding a trailing space in `EXPORT_ENV`